public/pages/ManagedIndices/containers/ManagedIndices/ManagedIndices.tsx

```diff
@@ -227,7 +247,7 @@ export default class ManagedIndices extends Component<ManagedIndicesProps, Manag
     try {
       const { managedIndexService, history } = this.props;
       const queryObject = ManagedIndices.getQueryObjectFromState(this.state);
-      const queryParamsString = queryString.stringify(queryObject);
+      const queryParamsString = queryString.stringify({ ...queryObject, dataSourceLabel: this.state.dataSourceLabel });
```

Comment:
dataSourceLabel would be part of the URL, even when MDS is not enabled.. I think the same pattern is being used in indices.tsx

Reply:
However, as we are deprecating dataSourceLabel, we should be good.